mongodump/prepare.go

```diff
@@ -149,7 +149,7 @@ func shouldSkipSystemNamespace(dbName, collName string) bool {
 			return true
 		}
 	case "config":
-		if collName == "transactions" || collName == "system.sessions" {
+		if collName == "transactions" || collName == "system.sessions" || collName == "transaction_coordinators" {
```

Comment:
Past 3 options, I think a switch/case block will be clearer, but this is fine for now.  :-)